src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+.page {
+  margin: 0;
```

Comment:
Be consistent with your margins. It's recommended to add margins only to the top or bottom of elements to avoid potential margin collapse issues. Please review if it's necessary to set the margin to 0 here, and if so, consider setting it only for the top or bottom.